docs/troubleshooting-development-issues.md

```diff
@@ -51,27 +51,12 @@ git clean -ifdX
 
 If you can't sign in, and instead you see a banner with an error message that it will be reported to freeCodeCamp, please double-check that your local port `3000` is not in use by a different program.
 
-<!-- tabs:start -->
-
-#### **macOS/Linux/WSL on Windows - From Terminal:**
```

Comment:
Leave the indication From Terminal here please

```
#### **From Terminal:**

Reply:
Sure, done 👍🏼